src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx

```diff
@@ -22,12 +26,12 @@ import { PlanDefinition, planStatusDisplay } from '../../../../configs/settings'
 import { HOME_URL, OPENSRP_PLANS, PLAN_LIST_URL, PLAN_UPDATE_URL } from '../../../../constants';
 import { displayError } from '../../../../helpers/errors';
 import { OpenSRPService } from '../../../../services/opensrp';
+import store from '../../../../store';
 import planDefinitionReducer, {
   fetchPlanDefinitions,
-  getPlanDefinitionsArray,
+  getPlanDefinitionsArrayByTitle,
```

Comment:
I wonder if it is possible to use `makePlansArraySelector` here?

Reply:
> can we please create an issue to address this ^^ please?

@moshthepitt What issue is this?

Reply:
@kelvin-muchiri the fact that when we get an empty list of plans we will continue showing a Loading indicator